sdk/servicebus/service-bus/test/perf/rhea-promise/receive.ts

```diff
@@ -11,13 +11,19 @@ Measures the maximum throughput of `receiver.receive()` in package `rhea-promise
 6. Example: `ts-node receive.ts 1000 1000000`
  */
```

Comment:
Can you please add the description referring to switching between auto vs manual credit management as a comment.